Dockerfile

```diff
@@ -1,6 +1,25 @@
 # Build the manager binary
 
-FROM registry.access.redhat.com/ubi8/go-toolset:1.22@sha256:780ab5f3874a6e2b1e04bb3719e614e835af3f8ab150922d6e84c2f9fd2bdb27 AS builder
+# BEGIN -- workaround lack of go-toolset for golang 1.23
+ARG GOLANG_IMAGE=golang:1.21
```

Comment:
Can you use `docker.io/library/golang:1.21` instead? The PR check doesn't work well with shortcuts.

Reply:
Btw, shouldn't it be 1:23?

Reply:
Thanks.  Yeah, I was obviously multi-tasking too much.  Fixed.